src/client/Microsoft.Identity.Client/Internal/Requests/SilentBrokerAuthStretegy.cs

```diff
@@ -1,48 +1,77 @@
 using System.Linq;
+using System.Text;
+using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Identity.Client.ApiConfig.Parameters;
+using Microsoft.Identity.Client.Cache;
+using Microsoft.Identity.Client.Cache.Items;
 using Microsoft.Identity.Client.Core;
-using Microsoft.Identity.Client.Internal.Requests;
+using Microsoft.Identity.Client.Internal.Broker;
 using Microsoft.Identity.Client.OAuth2;
 using Microsoft.Identity.Client.Utils;
 
-namespace Microsoft.Identity.Client.Internal.Broker
+namespace Microsoft.Identity.Client.Internal.Requests
 {
-    internal class BrokerSilentRequest
+    internal class SilentBrokerAuthStretegy : ISilentAuthStrategy
     {
-        public Dictionary<string, string> BrokerPayload
-            = new Dictionary<string, string>();
-        internal IBroker Broker { get; }
+        internal AuthenticationRequestParameters _authenticationRequestParameters;
+        private ICacheSessionManager CacheManager => _authenticationRequestParameters.CacheSessionManager;
+        protected IServiceBundle _serviceBundle;
         private readonly AcquireTokenSilentParameters _silentParameters;
-        private readonly AuthenticationRequestParameters _authenticationRequestParameters;
-        private readonly IServiceBundle _serviceBundle;
-        private readonly ICoreLogger _logger;
+        private SilentRequest _silentRequest;
+        private IBroker _broker;
+        public Dictionary<string, string> BrokerPayload = new Dictionary<string, string>();
+        ICoreLogger _logger;
 
-        internal BrokerSilentRequest(
-            AuthenticationRequestParameters authenticationRequestParameters,
-            AcquireTokenSilentParameters acquireTokenSilentParameters,
+        public SilentBrokerAuthStretegy(
+            SilentRequest request,
             IServiceBundle serviceBundle,
-            IBroker broker)
+            AuthenticationRequestParameters authenticationRequestParameters,
+            AcquireTokenSilentParameters silentParameters)
         {
             _authenticationRequestParameters = authenticationRequestParameters;
-            _silentParameters = acquireTokenSilentParameters;
+            _silentParameters = silentParameters;
             _serviceBundle = serviceBundle;
-            Broker = broker;
-            _logger = _authenticationRequestParameters.RequestContext.Logger;
+            _silentRequest = request;
+            _broker = _serviceBundle.PlatformProxy.CreateBroker(null);
+            _logger = authenticationRequestParameters.RequestContext.Logger;
+        }
+
+        public async Task<AuthenticationResult> ExecuteAsync(CancellationToken cancellationToken)
+        {
+            MsalAccessTokenCacheItem cachedAccessTokenItem = null;
+
+            if (!_silentParameters.ForceRefresh && !_authenticationRequestParameters.HasClaims)
+            {
+                cachedAccessTokenItem = await CacheManager.FindAccessTokenAsync().ConfigureAwait(false);
```

Comment:
You need to invoke the broker here, this is the `SilentBrokerAuth` strategy, not the `client` / `cache` strategy.